scripts/install_quickstarter.sh

```diff
@@ -12,6 +13,6 @@ ks generate kubebench-quickstarter-volume kubebench-quickstarter-volume
 
 ks apply ${KB_ENV} -c kubebench-quickstarter-service
 
-KB_NFS_IP=`kubectl get svc kubebench-nfs-svc -o=jsonpath={.spec.clusterIP}`
+KB_NFS_IP=`kubectl get svc kubebench-nfs-svc -o=jsonpath={.spec.clusterIP} -n ${K8S_NAMESPACE}`
```

Comment:
can we get the namespace programmably (e.g. through ks env describe?) instead of creating a separate namespace environment variable, the reason is if your k8s_namespace is different from whats set in your ks env, then this is not going to work.

Reply:
Done